packages/dashboard/src/app/views/exploreTemplates/index.js

```diff
@@ -191,8 +199,15 @@ function ExploreTemplates() {
       setActiveTemplate(newTemplate);
       setActiveTemplateIndex(offset);
       replace(`?id=${newTemplate.id}&isLocal=${newTemplate.isLocal}`);
+      speak(
+        sprintf(
+          /* translators: %s: template title */
+          __('Viewing %s', 'web-stories'),
+          newTemplate?.title
```

Comment:
`newTemplate` should always be defined since we are declaring it up above, the optional chaining shouldn't be necessary.